src/sample_weight_audit/estimator_check.py

```diff
@@ -172,12 +172,34 @@ def compute_predictions(est, X):
             return est.predict_proba(X)
         else:
             return est.decision_function(X)
+    elif isinstance(est, Pipeline):
+        return est.predict(X)
```

Comment:
Is this really needed? I would have expected `is_regressor(est)` to return `True`.